### PR TITLE
Merge working gossip code into dev 

### PR DESCRIPTION
**Pull request by easye**
* completed migration to TCP
* configuration code works 
* tested with both LispWorks and CCL

### CODE REVIEW

File: src/Actors/actors-startup.lisp [hunk 1]
```diff
@@ -214,16 +216,17 @@ THE SOFTWARE.
   (install-actor-directory)
   (install-actor-printer))
 
-#+(or :ALLEGRO :OPENMCL)
-(install-actor-system)
+#+(or :LISPWORKS :ALLEGRO :OPENMCL)
```
Comment (easye) [inline src/Actors/actors-startup.lisp:219]: Why is there a conditional?  Shouldn't this be for all platforms?

File: src/gossip/socket-actors-test.lisp [hunk 2]
```diff
@@ -0,0 +1,55 @@
+;;; socket-actor-test.lisp
```
Comment (easye) [inline src/gossip/socket-actors-test.lisp:1]: @svspire Maybe a comment to mark status as not working.

**Comment by easye** [comment]
Copy config files to 
```
(asdf:system-relative-pathname :emotiq "../var/etc/gossip-config/")
```

File: src/Actors/actors-startup.lisp (hunk 1 above)
Comment (svspire) [reply src/Actors/actors-startup.lisp:219]: I'm going to leave the conditional here for now until we can ensure that all Lispworks images have multiprocessing enabled. The possibility that that isn't happening is why the old conditional for lispworks is still present but commented out.

**Review by svspire — APPROVED** [review]
Travis deploy tests are failing because they seem to be using a non-multiprocessing version of Lispworks. I'm going to go ahead and approve and merge with dev, because the fix for Travis is outside this code.

**Pull request merged**